UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>; instead of , before "we will"</p><p> </p><p>A bit more explanation will help here, since there are 20 lines of code coming up. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>